.circleci/config.yml

```diff
@@ -37,14 +37,14 @@ jobs:
       - checkout
       - run:
           name: Fetch System Dependenciesu
-          command: HOMEBREW_NO_AUTO_UPDATE=1 brew install pcre luarocks
+          command: HOMEBREW_NO_AUTO_UPDATE=1 brew install pcre lua@5.1 luarocks
       - run:
           name: Download SDK
           command: ./download.sh
       - run:
           name: Build SDK
-          command: luarocks make launchdarkly-server-sdk-1.0-0.rockspec LD_DIR=. LD_INCDIR=./include
+          command: luarocks --lua-version=5.1 make launchdarkly-server-sdk-1.0-0.rockspec LD_DIR=. LD_INCDIR=./include
       - run:
           name: Run hello
           command: |
-            LD_LIBRARY_PATH=. lua5.3 hello.lua
```

Comment:
Nothing was installing lua, or specifically 5.3, so this failed. I used 5.1 for each step for now.

Reply:
Linux was fine, because usbuntu 18.04 will default to a compatible version of lua.